crates/sui-cost/src/empirical_transaction_cost.rs

```diff
@@ -31,6 +31,7 @@ pub enum CommonTransactionCosts {
     Publish,
     MergeCoin,
     SplitCoin(usize),
+    SplitCoinEqual,
```

Comment:
You should extend this to take the number of coins, and provide a few runs like the SplitCoin case.
This helps us track multiple data points

Reply:
Thanks @aschran. For more context on why we want multiple points:
- if we find that the cost grows fairly linearly with the number of splits, we can build formula-based cost estimator which allows apps and clients to quickly estimate what their splits might cost at any arbitrary N.